### ISSUE TITLE
Players aren't removed upon extraction

### ISSUE THREAD

**Issue by TehFl0w** [paulov-t/SIT.Core#190]
Extracted players are not yet removed from the world for other clients, not really an issue but something I noticed

**Comment by imonfireirl** [comment]
I just had a raid on Factory and got extract camped and killed by a bot, but after loading into another Factory I couldn't get into the Gate 3 extract as my dead body from the last match was blocking the doorway. I'm assuming this has might be related to your problem as well.

**Comment by KoviUbi62** [comment]
![Képernyőfelvétel (2045)](https://github.com/paulov-t/SIT.Core/assets/137239366/3e292ec6-053e-49de-af32-40ac71009e22)
Previous gear reamines when someone else dies or extraxts and it remains there till a game restart.

**Comment by ezra-guzman** [comment]
Yeah this also persists across raids in the same location from my experience. Shooting the floating armor of a player from a previous raid will kill the currently-living instance of that player, which I found the hard way haha
